debian/mariadb-server.postinst

```diff
@@ -191,9 +193,13 @@ EOF
     # handle things.
     # Debian: beware of the bashisms...
     # Debian: can safely run on upgrades with existing databases
+    # Workaround for Debian Bug #1022994: failure to create database when
+    # working with libpam-tmpdir (by setting TMPDIR to empty value).
```

Comment:
there's a mariadb bug for this. A long as its referenced in commit message, or otherwise closed with this merge its ok.

Reply:
Can you point JIRA bug so this as I can't find it

Reply:
https://jira.mariadb.org/browse/MDEV-29910